playground/index.html

```diff
@@ -15,6 +15,19 @@
             <button id="greenflag">Green flag</button>
             <button id="stopall">Stop</button>
         </div>
+        <div>
+            Turbo: <input id='turbomode' type='checkbox' />
+        </div>
+        <div>
+            Pause: <input id='pausemode' type='checkbox' />
+        </div>
+        <div>
+            Compatibility (30 TPS):  <input id='compatmode' type='checkbox' />
+        </div>
+        <div>
+            Single stepping:  <input id='singlestepmode' type='checkbox' />
```

Comment:
Also two spaces here?

Just nitpicking. It's not like HTML generally cares about multiple whitespace, but still..